var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -30,16 +30,17 @@ class SuiteSparse(Package):
     SuiteSparse is a suite of sparse matrix algorithms
     """
     homepage = 'http://faculty.cse.tamu.edu/davis/suitesparse.html'
-    url = 'http://faculty.cse.tamu.edu/davis/SuiteSparse/SuiteSparse-4.5.1.tar.gz'
+    url = 'http://faculty.cse.tamu.edu/davis/SuiteSparse/SuiteSparse-4.5.1.tar.gz'  # noqa: E501
 
     version('4.5.5', '0a5b38af0016f009409a9606d2f1b555')
     version('4.5.4', 'f6ab689442e64a1624a47aa220072d1b')
     version('4.5.3', '8ec57324585df3c6483ad7f556afccbd')
     version('4.5.1', 'f0ea9aad8d2d1ffec66a5b6bfeff5319')
 
     variant('tbb',  default=False, description='Build with Intel TBB')
-    variant('pic',  default=True,  description='Build position independent code (required to link with shared libraries)')
+    variant('pic',  default=True,  description='Build position independent code (required to link with shared libraries)')  # noqa: E501
     variant('cuda', default=False, description='Build with CUDA')
+    variant('openmp', default=True, description='Build with OpenMP')
```

Comment:
I would make it `default=False`. This will be consistent with all the other packages using the `openmp` variant. The rationale is that in the DAG of an OpenMP multi-threaded application only one node may need the `openmp` variant turned on (well, at least in 99% of the cases).